code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -14,10 +14,10 @@
 	var/shapeshift_type
 	var/list/current_shapes = list()
 	var/list/current_casters = list()
-	var/list/possible_shapes = list(/mob/living/simple_animal/mouse,\
-		/mob/living/simple_animal/pet/dog/corgi,\
-		/mob/living/simple_animal/hostile/carp/ranged/chaos,\
-		/mob/living/simple_animal/bot/ed209,\
+	var/list/possible_shapes = list(/mob/living/simple_animal/mouse,
+		/mob/living/simple_animal/pet/dog/corgi,
+		/mob/living/simple_animal/hostile/carp/ranged/chaos,
+//		/mob/living/simple_animal/bot/ed209,
```

Comment:
You seem to be misinterpreting my mental state as well as my point. You are phrasing your counterpoint in a manner that implies that you believe wizard as a gamemode should not occur on low pop. Your stated issue with the spell is it requires large numbers of players to make it balanced. The easiest way to handle that is to raise the population requirement from its current.. I think 10? to 45-50 similar to nukie/blob. It's a gamemode with a certain number of players in mind. Why not simply ensure that it only happens with that number of players?